housing_cooperative_base/models/building.py

```diff
@@ -54,7 +54,11 @@ class Building(models.Model):
         required=False,
     )
 
-    nb_rooms = fields.Integer(string="Number of Rooms", required=False)
+    nb_rooms = fields.Float(
```

Comment:
Shouldn't this be computed?

Reply:
These rooms are not the same as the `rooms` records, which refer to additional rooms that can be rented. nb_rooms just lists the number of rooms (kitchen, toilet, ...) inside a building. See 'Nombre de pièces dans l’immeuble' in spec.